nix/shells.nix

```diff
@@ -32,9 +38,7 @@ let
     };
 
     # for 'make watchman-clean'
-    watchman = mkShell {
-      buildInputs = with pkgs; [ watchman ];
-    };
+    watchman = mkShell { buildInputs = with pkgs; [ watchman ]; };
```

Comment:
Please don't change formatting in Nix files.

Reply:
This one is fixed.